src/memray/_memray/tracking_api.cpp

```diff
@@ -32,12 +33,38 @@ get_executable()
     return std::string(buff, len);
 }
 
+bool
```

Comment:
```suggestion
static inline bool
```

Reply:
OK, sure. I don't think this matters much, though - this is in an unnamed namespace, so `static` doesn't really have any effect, and given that we're using LTO, there's no chance this wasn't already being inlined.

Doesn't hurt to be explicit, though.